src/pms/factors/defaults.py

```
@@ -0,0 +1,43 @@
+
+
+def _step(
+    factor_id: str,
+    *,
+    role: str,
+    weight: float,
+    threshold: float | None = None,
+    param: str = "",
+) -> FactorCompositionStep:
+    projections_module = importlib.import_module("pms.strategies.projections")
+    step_cls = getattr(projections_module, "FactorCompositionStep")
```

Ruff (0.15.10)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)